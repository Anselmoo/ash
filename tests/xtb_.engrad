#
# Number of atoms
#
         3
#
# The current total energy in Eh
#
     -5.768502690905
#
# The current gradient in Eh/bohr
#
      -0.000000000000
       0.000000000000
      -0.012384849474
       0.000000000000
       0.001814240617
       0.006192424737
       0.000000000000
      -0.001814240617
       0.006192424737
#
# The atomic numbers and current coordinates in Bohr
#
   8    -2.6033361    0.0000000   -3.2885006
   1    -2.6033361    1.4349388   -2.1621426
   1    -2.6033361   -1.4349388   -2.1621426
