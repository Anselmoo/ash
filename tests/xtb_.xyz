3
title
O     -1.377626260000   0.000000000000  -1.740199718000
H     -1.377626260000   0.759337000000  -1.144156718000
H     -1.377626260000  -0.759337000000  -1.144156718000
