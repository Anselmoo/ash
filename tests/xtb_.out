      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.5.1 (conda-forge) compiled by 'runner@Mac-1657629245115.local' on 2022-07-12

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * C. Bannwarth, E. Caldeweyher, S. Ehlert, A. Hansen, P. Pracht,
     J. Seibert, S. Spicher, S. Grimme, WIREs Comput. Mol. Sci., 2020, 11,
     e01493. DOI: 10.1002/wcms.1493
   
   for GFN2-xTB:
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   for GFN1-xTB:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   for GFN0-xTB:
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   for GFN-FF:
   * S. Spicher and S. Grimme, Angew. Chem. Int. Ed., 2020, 59, 15665-15673.
     DOI: 10.1002/anie.202004239
   
   for ALPB and GBSA implicit solvation:
   * S. Ehlert, M. Stahn, S. Spicher, S. Grimme, J. Chem. Theory Comput.,
     2021, 17, 4250-4261. DOI: 10.1021/acs.jctc.1c00471
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   * E. Caldeweyher, J.-M. Mewes, S. Ehlert and S. Grimme, Phys. Chem. Chem. Phys.
     2020, 22, 8499-8512. DOI: 10.1039/D0CP00502A
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   * J. Koopman and S. Grimme, ACS Omega 2019, 4, 12, 15120-15133.
     DOI: 10.1021/acsomega.9b02011
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   for SPH calculations refer to:
   * S. Spicher and S. Grimme, J. Chem. Theory Comput., 2021, 17, 1701-1714
     DOI: 10.1021/acs.jctc.0c01306
   
   with help from (in alphabetical order)
   P. Atkinson, C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher
   M. Checinski, S. Dohm, S. Ehlert, S. Ehrlich, I. Gerasimov, C. Hölzer
   A. Katbashev, J. Koopman, C. Lavigne, S. Lehtola, F. März, M. Müller,
   F. Musil, H. Neugebauer, J. Pisarek, C. Plett, P. Pracht, F. Pultar,
   J. Seibert, P. Shushkov, S. Spicher, M. Stahn, M. Steiner, T. Strunk,
   J. Stückrath, T. Rose, and J. Unsleber
   
 * started run on 2022/12/30 at 17:27:21.139     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : /Users/bjornsson/miniforge3/envs/ASHv2/bin/xtb xtb_.xyz --gfn 1 --grad --chrg 0 --uhf 0 --iterations 500 --etemp 300 --acc 0.1 --parallel 1 --input xtbinput 
          coordinate file            : xtb_.xyz
          omp threads                :                     1

   ID    Z sym.   atoms
    1    8 O      1
    2    1 H      2, 3

           -------------------------------------------------
          |                 G F N 1 - x T B                 |
           -------------------------------------------------

        Reference                      10.1021/acs.jctc.7b00118
      * Hamiltonian:
        H0-scaling (s, p, d)           1.850000    2.250000    2.000000
        zeta-weighting                 0.000000
      * Dispersion:
        s8                             2.400000
        a1                             0.630000
        a2                             5.000000
        s9                             0.000000
      * Repulsion:
        kExp                           1.500000
        rExp                           1.000000
      * Coulomb:
        alpha                          2.000000
        third order                    atomic
        anisotropic                    false
      * Halogen bond correction:
        rad-scale                      1.300000
        damping                        0.440000

q/qsh data taken from xtbrestart

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                   8          :
          :  # atomic orbitals                   8          :
          :  # shells                            6          :
          :  # electrons                         8          :
          :  # halogen bonds                     0          :
          :  max. iterations                   500          :
          :  Hamiltonian                  GFN1-xTB          :
          :  restarted?                       true          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    0.1000000          :
          :  -> integral cutoff          0.3500000E+02      :
          :  -> integral neglect         0.1000000E-08      :
          :  -> SCF convergence          0.1000000E-06 Eh   :
          :  -> wf. convergence          0.2000000E-05 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1     -5.8038158 -0.580382E+01  0.528E-10    9.28       0.0  T
   2     -5.8038158 -0.799361E-14  0.171E-10    9.28  100000.0  T
   3     -5.8038158  0.266454E-14  0.738E-12    9.28  100000.0  T

   *** convergence criteria satisfied after 3 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.7580400             -20.6273
         2        2.0000           -0.6099979             -16.5989
         3        2.0000           -0.5496329             -14.9563
         4        2.0000           -0.5002420             -13.6123 (HOMO)
         5                         -0.1593787              -4.3369 (LUMO)
         6                         -0.0512735              -1.3952
         7                          0.3179238               8.6511
         8                          0.4506545              12.2629
      -------------------------------------------------------------
                  HL-Gap            0.3408633 Eh            9.2754 eV
             Fermi-level           -0.3298104 Eh           -8.9746 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.001 sec
 SCC setup                      ...        0 min,  0.000 sec (  4.757%)
 Dispersion                     ...        0 min,  0.000 sec (  1.743%)
 classical contributions        ...        0 min,  0.000 sec (  0.612%)
 integral evaluation            ...        0 min,  0.000 sec ( 17.004%)
 iterations                     ...        0 min,  0.000 sec ( 48.422%)
normal termination of xtb
 molecular gradient             ...        0 min,  0.000 sec ( 23.316%)
 printout                       ...        0 min,  0.000 sec (  3.533%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -5.768502689127 Eh    ::
         :: gradient norm              0.015383873390 Eh/a0 ::
         :: HOMO-LUMO gap              9.275363271180 eV    ::
         ::.................................................::
         :: SCC energy                -5.803815828794 Eh    ::
         :: -> electrostatic           0.050223102258 Eh    ::
         :: repulsion energy           0.035450733004 Eh    ::
         :: dispersion energy         -0.000137593337 Eh    ::
         :: halogen bond corr.         0.000000000000 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :: total charge               0.000000000000 e     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.7580400             -20.6273
         2        2.0000           -0.6099979             -16.5989
         3        2.0000           -0.5496329             -14.9563
         4        2.0000           -0.5002420             -13.6123 (HOMO)
         5                         -0.1593787              -4.3369 (LUMO)
         6                         -0.0512735              -1.3952
         7                          0.3179238               8.6511
         8                          0.4506545              12.2629
      -------------------------------------------------------------
                  HL-Gap            0.3408633 Eh            9.2754 eV
             Fermi-level           -0.3298104 Eh           -8.9746 eV

  Mulliken/CM5 charges         n(s)   n(p)   n(d)
     1O    -0.66625 -0.99608   1.697  4.969  0.000
     2H     0.33313  0.49804   0.667  0.000  0.000
     3H     0.33313  0.49804   0.667  0.000  0.000

Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom

 ---------------------------------------------------------------------------
     #   Z sym  total        # sym  WBO       # sym  WBO       # sym  WBO
 ---------------------------------------------------------------------------
     1   8 O    1.791 --     3 H    0.896     2 H    0.896
     2   1 H    0.896 --     1 O    0.896
     3   1 H    0.896 --     1 O    0.896
 ---------------------------------------------------------------------------

Topologies differ in total number of bonds
Writing topology from bond orders to xtbtopo.mol

 dipole moment from electron density (au)
     X       Y       Z   
  -0.0000  -0.0000   1.1291  total (Debye):    2.870


           -------------------------------------------------
          | TOTAL ENERGY               -5.768502689127 Eh   |
          | GRADIENT NORM               0.015383873390 Eh/α |
          | HOMO-LUMO GAP               9.275363271180 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2022/12/30 at 17:27:21.141     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.002 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.002 sec
 * ratio c/w:     1.000 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.001 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.001 sec
 * ratio c/w:     1.000 speedup

